packages/vuetify/src/mixins/activatable/index.ts

```diff
@@ -50,18 +54,27 @@ export default baseMixins.extend({
     if (slotType && ['v-slot', 'normal'].includes(slotType)) {
       consoleError(`The activator slot must be bound, try '<template v-slot:activator="{ on }"><v-btn v-on="on">'`, this)
     }
+
+    this.getActivator()
+  },
+
+  beforeDestroy () {
+    this.removeActivatorEvents()
   },
 
   methods: {
-    getValueProxy (): object {
-      const self = this
-      return {
-        get value () {
-          return self.isActive
-        },
-        set value (isActive: boolean) {
-          self.isActive = isActive
-        },
+    addActivatorEvents () {
+      if (
+        !this.activator ||
+        this.disabled ||
+        !this.activatorElement
+      ) return
+
+      const listeners = this.genActivatorListeners()
+      const keys = Object.keys(listeners)
+
+      for (const key of keys) {
+        (this.activatorElement as any)[`on${key}`] = listeners[key]
```

Comment:
addEventListener?

Reply:
There was a specific reason for this, I'll double check as to why.